Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "abci"
-version = "0.6.4"
+version = "0.6.5"
```

Comment:
as the protobuf definitions changed, it should be `0.7.0`? should it go with https://github.com/tendermint/rust-abci/pull/112 ?

Reply:
ah these changes will come when https://github.com/tendermint/tendermint/pull/4304 is merged, so you can ignore it for now. I was demoing and testing out how things would change for developers who are fetching the files